usaspending_api/awards/models/transaction_fabs.py

```diff
@@ -93,6 +93,16 @@ class TransactionFABS(models.Model):
     uri = models.TextField(blank=True, null=True, db_index=True)
     submission_id = models.IntegerField(blank=True, null=True)
     unique_award_key = models.TextField(null=True, db_index=True)  # From broker.
+    officer_1_name = models.TextField(null=True, blank=True)
+    officer_1_amount = models.DecimalField(max_digits=23, decimal_places=2, blank=True, null=True)
+    officer_2_name = models.TextField(null=True, blank=True)
+    officer_2_amount = models.DecimalField(max_digits=23, decimal_places=2, blank=True, null=True)
+    officer_3_name = models.TextField(null=True, blank=True)
+    officer_3_amount = models.DecimalField(max_digits=23, decimal_places=2, blank=True, null=True)
+    officer_4_name = models.TextField(null=True, blank=True)
+    officer_4_amount = models.DecimalField(max_digits=23, decimal_places=2, blank=True, null=True)
+    officer_5_name = models.TextField(null=True, blank=True)
+    officer_5_amount = models.DecimalField(max_digits=23, decimal_places=2, blank=True, null=True)
 
```

Comment:
Is there any reason we don't normalize this and the fields in fpds to a shared object?

Reply:
In hindsight, I can think of a couple good reasons, probably the first one being DAIMS rules.

Reply:
Django almost supports shared object in concept but in practice it requires class inheritance. It might be a confusing process and the final state won't be that clean.

Personally I like having all of the DB columns in one list even if they are sometimes repeated.

Reply:
Yeah.  I go back and forth on this myself.  On the one hand, using mixins makes the core of the models more concise and practically guarantees conformity.  On the other hand, now I have to go tracing through a bunch of mixins and super classes to figure out what fields a model is supposed to have and what they look like.  Plus you almost always end up needed to make minor field tweaks which causes a proliferation of the original mixins with minor variances.